includes/plugins/class-wordpress-seo.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Yoast (wordpress-seo) integration class.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Main class.
+ */
+class WordPress_SEO {
+	/**
+	 * Initialize hooks and filters.
+	 */
+	public static function init() {
+		add_filter( 'wpseo_enhanced_slack_data', [ __CLASS__, 'use_cap_for_slack_preview' ] );
+		add_filter( 'wpseo_opengraph_url', [ __CLASS__, 'http_ogurls' ] );
+		add_filter( 'wpseo_robots_array', [ __CLASS__, 'allow_indexing_guest_author_archive' ], 10, 2 );
```

Comment:
It works for me. But is this specific to Newspack? Shouldn't we do the fix in CAP itself?